src/solvers/dgmulti/sbp.jl

```diff
@@ -405,23 +455,15 @@ function calc_surface_integral!(du, u, surface_integral::SurfaceIntegralWeakForm
   nothing
 end
 
-function create_cache(mesh::DGMultiMesh, equations, dg::DGMultiFluxDiffPeriodicFDSBP, RealT, uEltype)
+function create_cache(mesh::DGMultiMesh, equations,
+                      dg::DGMultiFluxDiffPeriodicFDSBP, RealT, uEltype)
 
-  rd = dg.basis
   md = mesh.md
 
-  # for use with flux differencing schemes
-  Qrst_skew = compute_flux_differencing_SBP_matrices(dg)
-
   # storage for volume quadrature values, face quadrature values, flux values
   nvars = nvariables(equations)
   u_values = allocate_nested_array(uEltype, nvars, size(md.xq), dg)
-
-  # dummy storage to allow for compatibility with DGMultiFluxDiffSBP
-  fluxdiff_local_threaded = [zeros(SVector{nvars, uEltype}, size(md.xq, 1))]
-
-  return (; md, Qrst_skew, u_values, fluxdiff_local_threaded,
-            invJ = inv.(md.J), inv_wq = inv.(rd.wq),)
```

Comment:
Why isn't this needed anymore?

Reply:
Because we specialize `calc_volume_integral!` for periodic FDSBP operators, and the specialization doesn't need these variables anymore. 

1. `fluxdiff_local_threaded` is used to store element-local RHS contributions
2. We avoid using `Qrst_skew` by using instead strong-form operators `M \ Q` where `M` is the mass matrix and `Q` is the skew-symmetric FD operator. This avoids storing two sets of sparse operators (one in the cache, one in `solver.basis.Drst`).